qemu/tests/block_resize.py

```diff
@@ -111,9 +113,16 @@ def compare_block_size(session, block_cmd, block_pattern):
         # We need shrink the disk in guest first, than in monitor
         if block_size < old_block_size and disk_update_cmd:
             session.cmd(disk_update_cmd[index])
-        tmp = vm.monitor.block_resize(data_image_dev, block_size)
+
+        vm.monitor.block_resize(data_image_dev, block_size)
+
         if need_reboot:
             session = vm.reboot(session=session)
+
+        # We need to rescan disk in diskpart after block_resize in windows
+        if disk_rescan_cmd:
+            session.cmd(disk_rescan_cmd)
```

Comment:
Hi seems we don't need rescan disk it we already reboot the guest. So maybe we can make it an elif branch here.